### PR TITLE
Profile update function

### PR DESCRIPTION
**Pull request by sun-anhnq**
Related Tickets : https://edu-redmine.sun-asterisk.vn/issues/27420

WHAT : Create profile update form for user

HOW: 
- Create profile_update html file, profile_update function in views.py and profile update form in forms.py
- Fix some css file and css code in html template

Evidence :
image

### CODE REVIEW

**Review by leminhquangb — APPROVED** [review]
approved

**Review by tuantuan98 — APPROVED** [review]
approved

**Comment by sun-anhnq** [comment]
ready

File: gogoedu/models.py [hunk 1]
```diff
@@ -6,13 +6,17 @@
 from django.contrib.auth.models import AbstractUser
 from django.utils.translation import gettext_lazy as _
 from django.urls import reverse
+from PIL import Image
+
 
 class Catagory(models.Model):
     name = models.CharField(max_length=255)
+    image = models.ImageField(default='images/lesson_pics/default.jpg', upload_to='images/lesson_pics')
```
Comment (thaodtd-0665) [inline gogoedu/models.py]: nên check hiển thị default, chứ ko nên upload default như này e nha

File: gogoedu/templates/gogoedu/myuser_update.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
+{% extends "base_generic.html" %}
```
Comment (thaodtd-0665) [inline gogoedu/templates/gogoedu/myuser_update.html:1]: check authentication cho xử lý update nha em

Comment (sun-anhnq) [reply gogoedu/templates/gogoedu/myuser_update.html:1]: html file:
image
views:
image

**Comment by sun-anhnq** [comment]
ready

**Review by thaodtd-0665 — APPROVED** [review]

**Pull request merged**